terraform/modules/data-share-service/sqs.tf

```diff
@@ -1,5 +1,11 @@
-module "data_processor_queue" {
+module "supplier_event_queue" {
```

Comment:
Has terraform said what it will do for these new queues? Will it complain about deleting old queues?

Reply:
It will just delete them. There are probably ways to make this zero downtime (add two new queues, keep the current one. Once the deployment is complete run an instance of the old task to pick anything up off the old queue. Remove the old queue in a separate deployment) but I don't think it's worth the hassle at the moment

Reply:
Just wanted to make sure it wouldn't just fail